pczt/CHANGELOG.md

```diff
@@ -8,9 +8,29 @@ and this library adheres to Rust's notion of
 ## [Unreleased]
 
 ### Added
-- `pczt::roles::redactor`
+- `pczt::common`:
+  - `Global::{tx_version, version_group_id, consensus_branch_id, expiry_height}`
+  - `Zip32Derivation::{seed_fingerprint, derivation_path}`
```

Comment:
I'm not actually sure these are exposed anywhere or used (`Zip32Derivation` is `pub(crate)` and AFAICT not exposed through any indirect getter). The Keystone firmware currently depends on equivalent getters in `orchard::pczt`; I'll check which of these new getters are actually still being used, and clean up any that are unnecessary.